libraries/lib-effects/PerTrackEffect.cpp

```diff
@@ -108,10 +108,7 @@ bool PerTrackEffect::ProcessPass(Instance &instance, EffectSettings &settings)
    };
 
    const bool multichannel = numAudioIn > 1;
-   auto range = multichannel
-      ? mOutputTracks->Leaders()
-      : mOutputTracks->Any();
-   range.VisitWhile( bGoodResult,
+   const auto &waveTrackVisitor =
```

Comment:
Is this even allowed, a reference to a right value ?

Reply:
I wrote an unnecessary but also harmless `&` there which I will remove before committing.

There is a rule in C++ that when an initializer is bound to a local reference variable, then then lifetime of the object constructed on the right hand side is extended until the reference variable name goes out of scope.